docs/reference/ilm/ilm-tutorial.asciidoc

```diff
@@ -282,14 +282,15 @@ DELETE /_index_template/timeseries_template
 [[manage-time-series-data-without-data-streams]]
 === Manage time series data without data streams
 
-Even though <<data-streams, data streams>> are a convenient way to scale
-and manage time series data, they are designed to be append-only. We recognise there
-might be use-cases where data needs to be updated or deleted in place and the
-data streams don't support delete and update requests directly,
-so the index APIs would need to be used directly on the data stream's backing indices.
-
-In these cases, you can use an index alias to manage indices containing the time series data
-and periodically roll over to a new index.
+Even though <<data-streams, data streams>> are a convenient way to scale and manage time series
+data, they are designed to be append-only. We recognise there might be use-cases where data needs to
+be updated or deleted in place and the data streams don't support delete and update requests
+directly, so the index APIs would need to be used directly on the data stream's backing indices. In
+these cases we still recommend using a data stream.
+
+If you frequently send multiple documents using the same `_id` expecting last-write-wins, you can
+use an index alias to manage indices containing the time series data and periodically roll over to a
+new index.
```

Comment:
Nit: does it make sense to explicitly say "use an index alias _instead of a data stream_" (or something along those lines)? Or do you think that's clear enough already? My thinking is that I want to avoid people thinking they can use a data stream and then configure an alias on top of that to tackle this use case.

Reply:
Good idea, I've changed this.